packages/pdf/src/RPA/PDF/keywords/model.py

```diff
@@ -0,0 +1,733 @@
+                    )
+                    self.write(s)
+                    for child in item:
+                        if self.figure:
+                            self.figure.set_item(item)
+                        render(child)
+                    self.write("</figure>\n")
+                    self.current_page.add_content(self.figure)
+                    self.figure = None
+            elif isinstance(item, LTTextLine):
+                self.write('<textline bbox="%s">\n' % bbox2str(item.bbox))
+                for child in item:
+                    render(child)
+                self.write("</textline>\n")
+            elif isinstance(item, LTTextBox):
+                wmode = ""
+
+                if isinstance(item, LTTextBoxVertical):
+                    wmode = ' wmode="vertical"'
+                s = '<textbox id="%d" bbox="%s"%s>\n' % (
+                    item.index,
+                    bbox2str(item.bbox),
+                    wmode,
+                )
+                box = TextBox(item.index, item.bbox, wmode)
+                self.write(s)
+                box.set_item(item)
+                self.current_page.add_content(box)
+                for child in item:
+                    render(child)
+                self.write("</textbox>\n")
+            elif isinstance(item, LTChar):
+                s = (
+                    '<text font="%s" bbox="%s" colourspace="%s" '
+                    'ncolour="%s" size="%.3f">'
+                    % (
+                        enc(item.fontname),
+                        bbox2str(item.bbox),
+                        item.ncs.name,
+                        item.graphicstate.ncolor,
+                        item.size,
+                    )
+                )
+                self.write(s)
+                self.write_text(item.get_text())
+                self.write("</text>\n")
+            elif isinstance(item, LTText):
+                self.write("<text>%s</text>\n" % item.get_text())
+            elif isinstance(item, LTImage):
+                if self.figure:
+                    self.figure.set_item(item)
+                if self.imagewriter is not None:
+                    name = self.imagewriter.export_image(item)
+                    self.write(
+                        '<image src="%s" width="%d" height="%d" />\n'
+                        % (enc(name), item.width, item.height)
+                    )
+                else:
+                    self.write(
+                        '<image width="%d" height="%d" />\n' % (item.width, item.height)
+                    )
+            else:
+                assert False, str(("Unhandled", item))
+
+        render(ltpage)
+
+    def close(self) -> Document:
+        self.write_footer()
+        return self.active_pdf_document
+
+
+class ModelKeywords(LibraryContext):
+    """Keywords for converting PDF document into specific RPA object model"""
+
+    @keyword
+    def convert(self, source_path: str = None) -> None:
```

Comment:
Should this be visible to users or done automatically? It also has a fairly generic name that very likely overlaps with user keywords or other libraries